### PR TITLE
fix: [NPM] add check for invalid /0 cidr

### PR DESCRIPTION
**Pull request by ck319**
**Reason for Change**:

This fixes an issue where we were not throwing an error for CIDRs whose IPs were not 0.0.0.0 and ended with /0 during translation. Any CIDR with [ip]/0 is invalid unless IP is 0.0.0.0. 

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [x] adds unit tests

**Notes**:
Add check for invalid /0 CIDRs

### CODE REVIEW

File: npm/pkg/dataplane/ipsets/ipsetmanager_test.go [hunk 1]
```diff
@@ -1546,6 +1546,11 @@ func TestValidateIPSetMemberIP(t *testing.T) {
 			ipblock: "2345:0425:2CA1:0000:0000:0567:5673:23b5 nomatch",
 			want:    false,
 		},
+		{
```
Comment (huntergregory) [inline npm/pkg/dataplane/ipsets/ipsetmanager_test.go:1549]: not sure if there's a test for valid /0 yet

**Review by huntergregory — COMMENTED** [review]
added a comment about the UT. Could you also add a short description of the current issue and how your PR fixes it?

File: npm/pkg/dataplane/ipsets/ipsetmanager_test.go (hunk 1 above)
Comment (ck319) [reply npm/pkg/dataplane/ipsets/ipsetmanager_test.go:1549]: updated

**Review by huntergregory — APPROVED** [review]

**Review by vakalapa — APPROVED** [review]

**Pull request merged**